src/Select.js

```diff
@@ -437,7 +440,7 @@ export default class Select extends Component<Props, State> {
       onChange ? () => onChange(newValue, { action }) : undefined
     );
   };
-  selectValue = (newValue: OptionType) => {
+  selectOption = (newValue: OptionType) => {
```

Comment:
You mentioned something about a `setOptions` or `selectOptions` (plural) here based on grouping functionality. While that'd be a good idea to support, most probably, I think it should have a single entry point like this one and the value could be `Array<OptionType>`. If you're selecting one, you just pass an array of one item etc.